config/config.go

```diff
@@ -111,7 +112,14 @@ func (conf *TerraformConfig) ValidateHooks() error {
 	allHooks := append(conf.GetBeforeHooks(), conf.GetAfterHooks()...)
 
 	for _, curHook := range allHooks {
-		if len(curHook.Execute) < 1 || curHook.Execute[0] == "" {
+		hasLoadEnvVars := false
```

Comment:
This var is extra. You can just check `curHook.LoadEnvVars` on line 122.

Reply:
Okay